test/Microsoft.DotNet.Compiler.Common.Tests/Tests.cs

```diff
@@ -5,7 +5,7 @@
 using Microsoft.DotNet.Tools.Test.Utilities;
 using Xunit;
 
-namespace Microsoft.DotNet.Cli.Compiler.Common
+namespace Microsoft.DotNet.Cli.Compiler.Common.Tests
 {
     public class Tests : TestBase
```

Comment:
Maybe `GivenACommonCompilerOptions` or something.

---
In reply to: [71824678](https://github.com/dotnet/cli/pull/3914#discussion_r71824678) [](ancestors = 71824678)